tests/artifact/test_json_artifact.py

```diff
@@ -0,0 +1,22 @@
+import bentoml
+from bentoml.artifact import JSONArtifact
+
+
+@bentoml.artifacts([JSONArtifact("hparams")])
+class ExampleServiceWithJSONArtifact(bentoml.BentoService):
+    pass
+
+
+def test_json_artifact_simple_service_round_trip(tmp_path):
+    service = ExampleServiceWithJSONArtifact()
+    # 'hparams' = hyperparameters used at training time.
+    service.pack("hparams", {
+        "lr": 1e-3,
+        "patience": 10,
+        "decay": 0.1,
+        "batch_size": 16
+    })
+    service.save_to_dir(str(tmp_path))
+    del service
+    new_service = bentoml.bundler.load(str(tmp_path))
+    assert isinstance(new_service.artifacts.hparams, (dict, list, int, float, str))
```

Comment:
just `assert isinstance(new_service.artifacts.hparams, dict)`? since it knows exactly what `new_service.artifacts.hparams` is here?

Reply:
could also do something like:
```python
params = {
        "lr": 1e-3,
        "patience": 10,
        "decay": 0.1,
        "batch_size": 16
    }
 ...
assert new_service.artifacts.hparams == params
```

Reply:
Oh, yeah. That's actually clearer! My thinking was more along the lines of 'all these options would be valid JSON'. But I say we go with your seccond suggestion.

Reply:
sounds good!